src/lib/containers/home_page/people/UserCardListGroups.Mobile.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import UserCardListRotate from '../../../containers/UserCardListRotate'
+import { FacetColumnValuesRequest } from '../../../utils/synapseTypes'
+import ExpandableContent from '../ExpandableContent'
+import { UserCardListGroupsProps } from './UserCardListGroups'
+
+export default function UserCardListGroupsMobile(props: UserCardListGroupsProps) {
+  const { columnName, facetValues, ...rest } = props
+  return (
+    <div>
+      {facetValues?.map((el, curIndex) => {
+            const facetValue = facetValues[curIndex]
+            const selectedFacet:FacetColumnValuesRequest = {
+              columnName,
+              facetValues:[facetValue],
+              concreteType: 'org.sagebionetworks.repo.model.table.FacetColumnValuesRequest',
+            }
+            const content = <UserCardListRotate key={`UserCardListGroup-Mobile-${curIndex}`}
```

Comment:
Mobile version must have n UserCardListRotates (since more than one ExpandableContent might be expanded).